source/develop/release-management/features/storage/incremental-backup.md

```diff
@@ -640,14 +671,18 @@ backups tasks. Use during backup to track and montior backup.
   - vm_id: UUID
   - creation_date: TIMESTAMP
 
-Add vm_backup_disk_map table. This table keeps the backup url for every
-disk. This url will be used instead of the image path on the host when
-creating an image transfer for a disk.
+Add vm_backup_disk_map table. This table keeps the backup url and the backup mode
+for every disk. The url will be used instead of the image path on the host when
+creating an image transfer for a disk. The backup mode is used to inform the user
+on the type of backup that was taken for the disk (`full`|`incremental`) so he will
+know what will be the requested context for downloading the backup disk extents
+using imageio API.
```

Comment:
The backup mode should also be used for the ticket "dirty" flag. ticket "dirty" should be set to true only if backup_mode is "incremental".

Reply:
Mentioned in the section that dedicated to backup_mode